server/controllers/boardController.js

```diff
@@ -0,0 +1,33 @@
+import Board from '../models/boardModel';
+import { buildResponse } from '../utils/helpers';
+import {} from '../models/issuesModel';
+
+const getBoardDetails = async (req, res) => {
+  const id = parseInt(req.params.id, 10);
+  if (typeof id !== 'number') {
+    res.status(400).send(buildResponse(false, `${req.params.id} should be a number`));
+  }
+  try {
+    let [board] = await Board.find({ id });
+    if (!board) {
+      res.status(400).send(buildResponse(false, `No board found with id :${id}`));
+    }
+    board = await Board.populate(board, { path: 'issues' });
+    if (board) {
+      const { lifecycles, issues } = board;
```

Comment:
Use select statement for getting particular fields instead of getting everything

Reply:
Agreed with Manish's suggestion!